packages/fxa-content-server/app/styles/_breakpoints.scss

```diff
@@ -40,10 +40,12 @@ $breakpoints: (
 @mixin min-width($breakpoint) {
   @include breakpoint-error-check($breakpoint) {
     @if $breakpoint == 'tablet' {
+      // This must be 0px, ignore the lint warning.
```

Comment:
@byrw sorry for being a couple of days late on seeing your update here. I should have clarified and asked you to add `// sass-lint:disable zero-unit` to these lines to actually prevent the warning from showing up when running `grunt lint`, but I just tried this quickly myself and I think due to line 43's length, it's not picking up the comment. If you have time to investigate this that would be great, otherwise I'll go ahead merge this early next week since a number of other warnings are taken care of with your PR. :)

Reply:
@LZoog thank you for clarifying my misunderstanding.

About the `// sass-lint:disable zero-unit`. According to the source code
https://github.com/aszwemin/sass-lint/blob/19df7ba0a58d38c1a16913a757a274f01e1f089f/lib/ruleToggler.js#L180
`// sass-lint:disable zero-unit`, does not work probably because the sass-lint use the column from where the comment started, as a starting point to disable the rule. 
Example:
```
p {
  margin-left: 0px; // sass-lint:disable zero-unit
  margin-right: 0px;
}
```
The disable will only work on margin-right, and ignore margin-left.